cli/commands/flags.go

```diff
@@ -59,6 +59,10 @@ var (
 		Name:  "json, j",
 		Usage: "print raw json -- don't pretty print",
 	}
+	path = cli.StringFlag{
```

Comment:
dont we already have one?

Reply:
@adrianosela I thought you meant change the name to have to separate path variables :'D